server/szurubooru/func/mime.py

```diff
@@ -55,14 +58,17 @@ def get_extension(mime_type: str) -> Optional[str]:
         "image/heic": "heic",
         "video/mp4": "mp4",
         "video/webm": "webm",
+        "video/x-flv": "flv",
         "application/octet-stream": "dat",
     }
     return extension_map.get((mime_type or "").strip().lower(), None)
 
-
 def is_flash(mime_type: str) -> bool:
     return mime_type.lower() == "application/x-shockwave-flash"
 
+def is_flv(mime_type: str) -> bool:
+    return mime_type.lower() == "video/x-flv"
```

Comment:
I think it would be better if we added this mime type to the `is_video()` function instead of creating a new function.